### PR TITLE
Start work on support new chord_services format for bentoV2; improve WES run sorting; add web service-info

### PR DESCRIPTION
**Pull request by davidlougheed**
* Support new `bento_services.json` format while maintaining backwards-compatibility
* Reduce reliance on `bento_services.json` data wherever possible
* Improve WES run sorting by not pushing formerly stuck runs to the top
* Add a web `/service-info` endpoint 
* Lets Katsu (v2.17+) do its own table ownership deletion
* Rewrites/fixes issues with notifications & related components

**Requires Katsu v2.17 with garbage collect changes.**

### CODE REVIEW

**Comment by davidlougheed** [comment]
TODO: support old service url

File: src/index.js [hunk 1]
```diff
@@ -18,13 +18,15 @@ import App from "./components/App";
 const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;
 export const store = createStore(rootReducer, composeEnhancers(applyMiddleware(thunkMiddleware)));
 
+const BENTO_URL_WITH_TRAILING_SLASH = BENTO_URL.endsWith("/") ? BENTO_URL : `${BENTO_URL}/`;
```
Comment (gsfk) [inline src/index.js:21]: fails for me on Bento2.8, BENTO_URL is null here so `endsWith()` crashes

Comment (davidlougheed) [reply src/index.js:21]: will fix, ty!

Comment (davidlougheed) [reply src/index.js]: should be fixed now

**Review by brouillette — APPROVED** [review]
Works on my end!

**Pull request merged**